### PR TITLE
Add Hive Services

### PR DESCRIPTION
**Pull request by KJonline**
**Description:**
Following the merging of home-assistant/home-assistant#26789 this is the documentation to support it with the below PR fixing wording.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#27038

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by KJonline** [comment]
Hi @frenck, I had issues trying to rebase the previous PR to get rid of the old commits out of it. the short story is I have created a different PR closing the other incorporating the changes you mentioned.

File: source/_components/hive.markdown [hunk 1]
```diff
@@ -57,6 +62,64 @@ scan_interval:
...
+  boost_hot_water:
+    sequence:
+      - service: "hive.boost_hot_water"
+        data:
+          entity_id: "water_heater.hot_water"
+          time_period: '01:30:00'
+          on_off: 'on
+```
+
+## Platforms
```
Comment (klaasnicolaas) [inline source/_components/hive.markdown]: I think we can skip the platform as section

File: source/_components/hive.markdown [hunk 2]
```diff
@@ -19,18 +20,22 @@ redirect_from:
...
 
 There is currently support for the following device types within Home Assistant:
 
-- [Binary Sensor](#Binary-Sensor)
-- [Climate](#Climate)
-- [Light](#Light)
-- [Sensor](#Sensor)
-- [Switch](#Switch)
-- [Water Heater](#Water-Heater)
+- [Services](#services)
```
Comment (klaasnicolaas) [inline source/_components/hive.markdown:29]: Service is not a device type 😉

File: source/_components/hive.markdown (hunk 1 above)
Comment (klaasnicolaas) [inline source/_components/hive.markdown:115]: I think we can leave the away

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_components/hive.markdown (hunk 2 above)
Comment (KJonline) [reply source/_components/hive.markdown:29]: I have updated the description to advise these are the services and platforms supported in home assistant.

File: source/_components/hive.markdown (hunk 1 above)
Comment (KJonline) [reply source/_components/hive.markdown:115]: Sorry I am not sure what you mean by leave the away?

Comment (klaasnicolaas) [reply source/_components/hive.markdown:115]: For now it's okay 😉

**Review by frenck — APPROVED** [review]
Thanks, @KJonline! 👍

**Pull request merged**